packages/modules/fulfillment/src/services/fulfillment-module-service.ts

```diff
@@ -1378,6 +1414,13 @@ export default class FulfillmentModuleService
               updatedRuleIds.push(rule.id)
             }
 
+            // @ts-ignore
+            delete rule.created_at
+            // @ts-ignore
+            delete rule.updated_at
+            // @ts-ignore
+            delete rule.deleted_at
```

Comment:
This is some this that needs more investigation. For now mikro orm doesn't handle those type properly for some reasons with our current config and we get something like this Data.value error which is very weird and only happen during commit but otherwise all the assign and validation passes first 🤔

Reply:
Alright, but we haven't had this issue before correct?

Reply:
yes it is new and I dont know why it is happening,I will try to investigate one more time but otherwise I would suggest we move forward for now wdyt?

Reply:
Yeah, agree – not worth spending much time on right now.